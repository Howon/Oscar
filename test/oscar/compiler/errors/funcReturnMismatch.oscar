def add5(int: x) => int = {
	return x + 5;
}

def main() => unit = {
	double y = add5(1));
}