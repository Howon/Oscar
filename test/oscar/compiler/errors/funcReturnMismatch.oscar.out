Return type mismatch: expected Int_t but returns Double_t